src/pcm.rs

```diff
@@ -923,6 +949,16 @@ impl Status {
     }
 }
 
+/// The type of timestamp returned by `Status::get_audio_htstamp()`.
+/// These values are derived from the [audio_time](https://github.com/alsa-project/alsa-lib/blob/master/test/audio_time.c)
+/// test program in the ALSA library.
+pub enum AudioTstampType {
```

Comment:
Hmm, but looking at the referenced file above, there are six different values (lines 272-283), not four. Could you explain?

Reply:
Good catch. Unfortunately I haven't been able to find any decent documentation for the values. I was going off the usage string, I'd expect live code to be more accurate. I'll change the enum to use those values.